### PR TITLE
Fix metric call

### PR DESCRIPTION
**Pull request by abarnw**
@kirillseva @tonglu @peterhurford 

The call order of the parameters used was incorrect resulting in garbage Gini and an nRMSD off by a scaling factor.

### CODE REVIEW

**Comment by tonglu** [comment]
LGTM

**Comment by kirillseva** [comment]
:+1:

**Comment by drbabar** [comment]
:+1:

**Comment by iveksl2** [comment]
This is a hotfix but if you want for thoroughness you can technically test this...
```
ret <- make_validation_plot(list(a = rnorm(100), b = rnorm(100)), data.frame(dep_var = rnorm(100)))
```
R-STABLE> ret$a$gini
[1] -0.1142988

**Comment by iveksl2** [comment]
:+1:

File: DESCRIPTION [hunk 1]
```diff
@@ -3,7 +3,7 @@ Type: Package
 Title: Hosmer-Lemeshow Test Plot (http://github.com/avantcredit/make_validation_plot)
 Description: R plotting tools for visualizing a Hosmer-Lemeshow
     goodness-of-fit test on a binary classifier.
-Version: 0.1.2
+Version: 0.1.3
```
Comment (peterhurford) [inline DESCRIPTION]: @abarnw @tonglu Please update `NEWS.md` with updates in version.

File: R/make_validation_plot.R [hunk 2]
```diff
@@ -404,10 +404,9 @@ make_validation_object.numeric <- function(scores, validation_data, buckets,
     # Definition:
     #   "nGini" reflects the relative order between y_act and y_pred, its value is bounded between [-1, 1]
     #           a perfect preservation of order will yield nGini = 1, and a total reverse of order will yield nGnini = -1
-    #   "nRMSD" is the most common metric for Regression Problems, a perfect prediction will yield nRMSD = 1
-    #           In practice the order should ~ 1, if you see nRMSD > 10, you should be worried
-    validation_object$NRMSD <- NormalizedRMSD(scores, validation_data[[dep_var_name]])
-    validation_object$gini  <- NormalizedGini(scores, validation_data[[dep_var_name]])
+    #   "nRMSD" is the most common metric for Regression Problems, a perfect prediction will yield nRMSD = 0
+    validation_object$NRMSD <- NormalizedRMSD(validation_data[[dep_var_name]], scores)
+    validation_object$gini  <- NormalizedGini(validation_data[[dep_var_name]], scores)
```
Comment (peterhurford) [inline R/make_validation_plot.R]: Yikes! @iveksl2 is right that we need tests for this.

**Pull request merged**